docs/data/profile-properties.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Profile Properties 
+description: Profile Properties allow you to sync a current property value from a source (currently only Snowflake) to use within the Amplitude platform.
+---
+
+## Context
+Customers can now bring in non-behavioral, customer profile data from their data warehouse to merge it with the existing behavioral product data already living in Amplitude. These values will always display the most current value being synced from the customer’s data warehouse.
+
+## Setup
+To set up a Profile Property in Amplitude, you must connect Amplitude to your data warehouse. Once connected, and you reach the data configuration section, you must select the “Warehouse Props” data type in the dropdown. From there, there are two minimum requirements for the import: a user identifier (user_id) and a profile property. Note: you may add more than one warehouse property per import, but there must be at least one per import.
+
+## Data Specifications
+The maximum number of warehouse properties currently supported for a single profile property import is 200. Today, profile properties are only supported for known Amplitude users. Therefore, each profile property must be accompanied by a user identifier (user_id).
```

Comment:
```suggestion
Profile Properties supports a maximum of 200 warehouse properties, and supports known Amplitude users. A `user_id` must accompany each profile property.
```

Present tense, active voice, we shouldn't date the article with "today" or "currently".